src/components/journeymap/JourneyMap.js

```diff
@@ -25,11 +25,7 @@ class JourneyMap extends React.Component {
     }
 
     render() {
-        if (!this.props.show) {
```

Comment:
et voilá

Reply:
I was trying to fix a problem with one of the tests, then forgot to remove this when I found another way to resolve it.